PictureApp/Controller/PreviewViewController.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  PreviewViewController.swift
+//  PictureApp
+//
+//  Created by Irina Gorbachenko on 04.12.2021.
+//
+
+import UIKit
+import NetworkLayer
+
+class PreviewViewController: UIViewController {
+
+    @IBOutlet private weak var imageViewPreview: UIImageView!
+    @IBOutlet var tagsLabel: UILabel!
+    
+    var selectedPicture: Post?
+    var tagText : String?
```

Comment:
Doesn't the post has information about tags?
I think you could just use selectedPost and extract tags from there.

Reply:
Fixed